config/locales/en.yml

```diff
@@ -141,237 +142,253 @@ en:
             options:
               option_yes:
-                label: "Yes"
+                label: 'Yes'
               option_may_lose:
-                label: "I do now but I might lose it"
+                label: I do now but I might lose it
               option_no:
-                label: "No"
+                label: 'No'
               not_sure:
-                label: "Not sure"
-            custom_select_error: "Select if you have somewhere to live"
+                label: Not sure
+            custom_select_error: Select if you have somewhere to live
           have_you_been_evicted:
-            title: "Have you been evicted?"
+            title: Have you been evicted?
             options:
               option_yes:
-                label: "Yes"
+                label: 'Yes'
               option_soon:
-                label: "I might be evicted soon"
+                label: I might be evicted soon
               option_no:
-                label: "No"
+                label: 'No'
               not_sure:
-                label: "Not sure"
-            custom_select_error: "Select if you have been evicted or might be soon"
+                label: Not sure
+            custom_select_error: Select if you have been evicted or might be soon
       mental_health:
-        title: "Mental health and wellbeing"
+        title: Mental health and wellbeing
         questions:
           mental_health_worries:
-            title: "Are you worried about either your mental health or someone else’s mental health?"
+            title: Are you worried about either your mental health or someone else’s
+              mental health?
             options:
               option_yes:
-                label: "Yes, I am"
+                label: Yes, I am
               option_no:
-                label: "No, I’m not"
+                label: No, I’m not
               not_sure:
-                label: "Not sure"
-            custom_select_error: "Select yes if you’re worried about your mental health or someone else’s mental health"
-      leave_home: # Q: Are you able to leave your home if absolutely neccessary?
+                label: Not sure
+            custom_select_error: Select yes if you’re worried about your mental health
+              or someone else’s mental health
+      leave_home:
         questions:
           able_to_leave:
-            title: "Are you able to leave your home for food, medicine, or health reasons?"
+            title: Are you able to leave your home for food, medicine, or health reasons?
             options:
               option_yes:
-                label: "Yes"
+                label: 'Yes'
               option_has_symptoms:
-                label: "I should not leave home because I have coronavirus symptoms, or someone in my household does"
+                label: I should not leave home because I have coronavirus symptoms,
+                  or someone in my household does
```

Comment:
Can this be split over two lines? I thought YAML needed a `|` before to do that without quotes?
```suggestion
                label: |
                  I should not leave home because I have coronavirus symptoms,
                  or someone in my household does
```

Reply:
Humm checked a rendered version of this:
Seems to be coming out ok, an dthat was the default that psych output.
![image](https://user-images.githubusercontent.com/3694062/82884593-d42fa300-9f3b-11ea-9f05-2e0693265345.png)

Reply:
Ah ha, so turns out you _can_ have plain:
https://yaml-multiline.info/

But:
> Note: Plain flow scalars are picky about the : and # characters. They can be in the string, but : cannot appear before a space or newline, and # cannot appear after a space or newline; doing this will cause a syntax error. If you need to use these characters you are probably better off using one of the quoted styles instead.

As these imports will now all be done automatically through the spreadsheet import, i think it's best that we just leave it to psych to choose the right one. I guess if we add a `:` into a string psych will switch to one of these pipe broken multiline yaml strings.